Source/WebKit/UIProcess/Cocoa/WKTextIndicatorStyleType.h

```diff
@@ -25,6 +25,8 @@
 
 #pragma once
 
+
```

Comment:
I implicitly meant that these extra newlines should be removed but I guess that wasn't quite obvious

Reply:
I got some of them, but I guess not all of them?